### PR TITLE
TELCODOCS-1014 - Detailing methods for adding worker nodes

### PR DESCRIPTION
**Pull request by rohennes**
[TELCODOCS-1014](https://issues.redhat.com//browse/TELCODOCS-1014): Need some signposting moule to direct users to the different ways to add worker nodes depending on their installation, environment, etc.

Version(s):
4.12+

Issue:
https://issues.redhat.com/browse/TELCODOCS-1014

Link to docs preview:
https://53515--docspreview.netlify.app/openshift-enterprise/latest/post_installation_configuration/cluster-tasks.html#adding-worker-nodes_post-install-cluster-tasks

QE review:
I don't think QE is required as this is conceptual info with no procedures.

Additional information:

### CODE REVIEW

File: modules/adding-worker-nodes.adoc [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+:_module-type: CONCEPT
+[id="adding-worker-nodes_{context}"]
+= Adding worker nodes
+
+After you deploy your OpenShift Container Platform cluster, you can add worker nodes to scale cluster resources. There are different ways you can add worker nodes depending on the installation method and the environment of your cluster.
+
+* For installer-provisioned infrastructure (IPI) clusters, you can manually or automatically scale the machineset to match the number of available bare metal hosts.
++ 
+To add a new bare metal host, you must configure all network prerequistites, configure an assoicated `baremetalhost` object, then provision the worker node to the cluster. You can add a new bare metal host manually or by using the web console.
```
Comment (elfosardo) [inline modules/adding-worker-nodes.adoc:16]: s/assoicated/associated

**Review by elfosardo — CHANGES_REQUESTED** [review]

File: modules/adding-worker-nodes.adoc (hunk 1 above)
Comment (rohennes) [reply modules/adding-worker-nodes.adoc]: @elfosardo - hey - I'm looking to get a LGTM on this PR so I can move it forward. Are you able to give a LGTM on some or all of the content? I know some might be outside of your remit. It's a bit tricky as it crosses teams.

**Comment by zaneb** [comment]
I'm not an expert on all of this methods, but this looks plausible to me. You can't go too far wrong by linking to the other available docs.

File: post_installation_configuration/cluster-tasks.adoc [hunk 2]
```diff
@@ -162,6 +162,47 @@ xref:../installing/installing_aws/installing-aws-network-customizations.adoc#ins
...
+
+=== Adding worker nodes to UPI clusters
+
+For user-provisioned infrastructure (UPI) clusters, you can add worker nodes using RHEL or RHCOS ISO images and connect it to your cluster using cluster ignition configs. For RHEL worker nodes, the following example uses ansible playbooks to add worker nodes to the cluster. For RHCOS worker nodes, the following example uses an ISO image and network booting to add worker nodes to the cluster. 
+
+* xref:../post_installation_configuration/node-tasks.adoc#post-install-config-adding-fcos-compute[Adding RHCOS worker nodes to a UPI cluster]
+
+* xref:../post_installation_configuration/node-tasks.adoc#post-install-config-adding-rhel-compute[Adding RHEL worker nodes to a UPI cluster]
+
+=== Adding worker nodes to AI clusters
```
Comment (dhellmann) [inline post_installation_configuration/cluster-tasks.adoc:190]: Maybe "Adding worker nodes to clusters managed by the assisted installer" would be clearer than using the acronym AI here?

**Review by dhellmann — COMMENTED** [review]
One suggestion for a heading change, which I'll leave up to you and the content team. Otherwise this looks good.

/lgtm

File: post_installation_configuration/cluster-tasks.adoc [hunk 3]
```diff
@@ -162,6 +162,47 @@ xref:../installing/installing_aws/installing-aws-network-customizations.adoc#ins
...
+
+For user-provisioned infrastructure clusters, you can add worker nodes by using a {op-system-base} or {op-system} ISO image and connecting it to your cluster using cluster ignition configs. For RHEL worker nodes, the following example uses Ansible playbooks to add worker nodes to the cluster. For RHCOS worker nodes, the following example uses an ISO image and network booting to add worker nodes to the cluster.
+
+* xref:../post_installation_configuration/node-tasks.adoc#post-install-config-adding-fcos-compute[Adding RHCOS worker nodes to a user-provisioned infrastructure cluster]
+
+* xref:../post_installation_configuration/node-tasks.adoc#post-install-config-adding-rhel-compute[Adding RHEL worker nodes to a user-provisioned infrastructure cluster]
+
+=== Adding worker nodes to clusters managed by the Assisted Installer
+
+For clusters managed by the Assisted Installer, you can add worker nodes by using the OpenShift Cluster Manager console, the Assisted Installer REST API or you can manually add worker nodes using an ISO image and cluster ignition configs.
```
Comment (bmcelvee) [inline post_installation_configuration/cluster-tasks.adoc:192]: s/OpenShift Cluster Manager/{cluster-manager-first}

File: post_installation_configuration/cluster-tasks.adoc [hunk 4]
```diff
@@ -162,6 +162,47 @@ xref:../installing/installing_aws/installing-aws-network-customizations.adoc#ins
...
+ 
+* xref:../scalability_and_performance/managing-bare-metal-hosts.adoc#adding-bare-metal-host-to-cluster-using-web-console_managing-bare-metal-hosts[Adding worker nodes using the web console] 
+
+* xref:../scalability_and_performance/managing-bare-metal-hosts.adoc#adding-bare-metal-host-to-cluster-using-yaml_managing-bare-metal-hosts[Adding worker nodes using YAML in the web console]
+
+* xref:../installing/installing_bare_metal_ipi/ipi-install-expanding-the-cluster.adoc#preparing-the-bare-metal-node_ipi-install-expanding[Manually adding a worker node to an installer-provisioned infrastructure cluster]
+
+=== Adding worker nodes to user-provisioned infrastructure clusters
+
+For user-provisioned infrastructure clusters, you can add worker nodes by using a {op-system-base} or {op-system} ISO image and connecting it to your cluster using cluster ignition configs. For RHEL worker nodes, the following example uses Ansible playbooks to add worker nodes to the cluster. For RHCOS worker nodes, the following example uses an ISO image and network booting to add worker nodes to the cluster.
```
Comment (bmcelvee) [inline post_installation_configuration/cluster-tasks.adoc:184]: Should configs be configuration, or is this standard terminology for cluster ignition?

File: post_installation_configuration/cluster-tasks.adoc [hunk 5]
```diff
@@ -162,6 +162,47 @@ xref:../installing/installing_aws/installing-aws-network-customizations.adoc#ins
 
 include::modules/images-update-global-pull-secret.adoc[leveloffset=+1]
 
+[id="adding-worker-nodes_{context}"]
+== Adding worker nodes
+
+After you deploy your OpenShift Container Platform cluster, you can add worker nodes to scale cluster resources. There are different ways you can add worker nodes depending on the installation method and the environment of your cluster.
```
Comment (bmcelvee) [inline post_installation_configuration/cluster-tasks.adoc:168]: s/OpenShift Container Platform/{product-title}

**Comment by bmcelvee** [comment]
Just a couple of small things. Otherwise, LGTM!

File: post_installation_configuration/cluster-tasks.adoc (hunk 4 above)
Comment (rohennes) [reply post_installation_configuration/cluster-tasks.adoc:184]: "Ignition config files" is best I think

**Pull request merged**